drift_dev/lib/src/analysis/resolver/dart/view.dart

```diff
@@ -188,11 +189,28 @@ class DartViewResolver extends LocalElementResolver<DiscoveredDartView> {
         ),
       );
     }
+    AnnotatedDartCode query;
+    if (resolvedFrom == null &&
+        resolver.driver.options.assumeCorrectReference) {
+      query = AnnotatedDartCode.build(
+          (builder) => builder.addText(body.expression.toSource().replaceAll(target!.toSource(), '')));
+    } else {
+      query = AnnotatedDartCode.build(
+          (builder) => builder.addAstNode(body.expression, exclude: {target!}));
+    }
 
-    final query = AnnotatedDartCode.build(
-        (builder) => builder.addAstNode(body.expression, exclude: {target!}));
+    // if(resolver.driver.options.assumeCorrectReference){
+    //   bool separate = false;
+    //   for (int i = 0; i < query.elements.length; i++) {
+    //     if (separate && query.elements[i] is String && i != query.elements.length-1) {
+    //       query.elements[i]+=',';
+    //       separate = false;
+    //     }
+    //     separate = query.elements[i] is DartTopLevelSymbol;
+    //   }
+    // }
```

Comment:
I don't understand what this did, is this solving another problem introduced by the option? Either way, if it's not used it should be removed.

Reply:
Done, was the first attempt as the joins did not add the separator "," . But then having a group by I could not use this simple for loop and resolved it with the current solution.